go/connector/run.go

```diff
@@ -304,15 +308,15 @@ func runCommand(
 			fe.onError(fmt.Errorf("go.estuary.dev/E116: connector failed, with error: %w\nwith stderr:\n\n%s",
 				waitErr, cmd.Stderr.(*connectorStderr).buffer.String()))
 		} else {
-			fe.onError(ctx.Err())
+			fe.onError(fmt.Errorf("ctx err: %w", ctx.Err()))
 		}
 	}
 	_ = stderrForwarder.Close()
 
 	if port != "" && conn != nil {
 		var closeErr = conn.Close()
 		if closeErr != nil {
-			fe.onError(closeErr)
+			fe.onError(fmt.Errorf("conn.Close: %w", closeErr))
```

Comment:
This could maybe be a little more specific to the part of the code that it's in. Don't know if there's other `conn.Close` errors out there. Not a big deal though.